eve/workers/end2end/Dockerfile

```
@@ -15,11 +16,13 @@ RUN apt-get update && apt-get install -y \
     git \
     gnupg2 \
     python3-buildbot-worker \
-    software-properties-common
+    software-properties-common \
+    m4
 
 # install main dependencies
 RUN curl -Lo /usr/local/bin/kind https://kind.sigs.k8s.io/dl/${KIND_VERSION}/kind-$(uname)-amd64 && \
     chmod +x /usr/local/bin/kind && \
+    curl https://raw.githubusercontent.com/helm/helm/${HELM_VERSION}/scripts/get-helm-3 | bash && \
```

Optional: I would rather avoid doing `curl | bash` commands when possible, suggesting the following changes:
```suggestion
  curl -sSL https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz | tar -xvz && \
    install linux-amd64/helm /usr/local/bin && \
    rm -rf linux-amd64 && \
```